scripts/solr/reindex-zac-solr-data.sh

```diff
@@ -1,14 +1,45 @@
-#!/bin/sh
+#!/bin/bash
 
 #
-# SPDX-FileCopyrightText: 2023 Lifely
+# SPDX-FileCopyrightText: 2024 Lifely
 # SPDX-License-Identifier: EUPL-1.2+
 #
 
```

Comment:
I think we need to turn on exit on error
```suggestion
set -e

```